### PR TITLE
Components

### PR DESCRIPTION
**Pull request by Vishnu-Aithal**
Documentation Page for Component Library ,
Individual Components are already reviewed in a separate repository.
Please Review the Layout and Documentation.

Preview - https://deploy-preview-1--some-ui.netlify.app/

### CODE REVIEW

**Review by ArjunJunna — APPROVED** [review]
Everything looks fine. You can merge it.

File: index.js [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+  // console.log(...onScreen);
+
+  // const targetId = onScreen && onScreen.pop();
+  // const targetLink = [...menuLinks].find(
+  //   (link) => link.href.split("#").pop() === targetId
+  // );
+
+  // targetLink.classList.add("side-bar__link--active");
+
+  //-------------------------------------------------------------------
```
Comment (foolhardy21) [inline index.js:46]: hey @Vishnu-Aithal , i think this can be removed.

**Review by foolhardy21 — APPROVED** [review]
ui is amazing.

File: index.js (hunk 1 above)
Comment (Vishnu-Aithal) [reply index.js:46]: ok will do it .. thanks

**Comment by Vishnu-Aithal** [comment]
Thanks For Reviewing !

**Pull request merged**